services/121-service/migration/1701426130489-CreateAndUpdateMessageTemplates.ts

```diff
@@ -8,7 +8,7 @@ export class CreateAndUpdateMessageTemplates1701426130489
   public async up(queryRunner: QueryRunner): Promise<void> {
     // Commit transaction because the tables are needed before the insert
     await queryRunner.commitTransaction();
-    await this.migrateData(queryRunner);
+    // await this.migrateData(queryRunner);
```

Comment:
This would  still need to happen the first time this code is deployed/run on any instance, right?

So we can only remove/comment-out this code AFTER it was part of a deployed release, right?

Reply:
@elwinschmitz yes, but this was already released right? (only relevant for NLRC)